examples/randomWalk.js

```diff
@@ -11,13 +11,33 @@ function stepFunc() {
     return pg.geometry.toCircle(2 * Math.PI * Math.random())
 }
 
+function evenCircleStart(numPoints, radius){
+    return pg.geometry.circlePoints(numPoints, radius);
+}
+
+function archSpiralStart(a, b, n, pointsPerCycle){
```

Comment:
This is a nice function to have, why don't we put it in pg.geometry (source/geometry/index.js)?